src/main/java/appeng/parts/misc/PartStorageBus.java

```diff
@@ -367,6 +368,41 @@ private void resetCache()
 		Platform.postListChanges( before, after, this, this.mySrc );
 	}
 
+	@SuppressWarnings( "unchecked" )
+	private IMEInventory<? extends IAEItemStack> getInventoryWrapper( TileEntity target )
+	{
+
+		EnumFacing targetSide = this.getSide().getFacing().getOpposite();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'targetSide' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)